react/src/pages/MapProject/MapProject.tsx

```diff
@@ -116,6 +117,21 @@ const MapProject: React.FC<MapProjectProps> = ({ isPublicView = false }) => {
       </div>
     );
   }
+  if (isPublicView && activeProject.public === false) {
+    return (
+      <div className={styles.root}>
+        <HeaderNavBar />
+        <div className={styles.errorContainer}>
+          <Message type="error" tagName="div">
+            <p>
+              This is not a public map. You are unauthorized to view this
```

Comment:
I think "This is not a public map" is sufficient.

This is catching the scenario where they are logged in and must be a member of the this map-project as the request for project worked.